src/components/views/rooms/BasicMessageComposer.js

```diff
@@ -370,6 +370,16 @@ export default class BasicMessageEditor extends React.Component {
         } else if (modKey && event.key === Key.GREATER_THAN) {
             this._onFormatAction("quote");
             handled = true;
+        // redo
+        } else if ((modKey && event.key === Key.Y) ||
+                  (IS_MAC && event.shiftKey && event.key === Key.Z)) {
```

Comment:
```suggestion
                  (IS_MAC && modkey && event.shiftKey && event.key === Key.Z)) {
```
...and looks like this isn't checking for cmd?